components/brave_wallet_ui/page/screens/composer_ui/select_token_modal/select_token_modal.tsx

```diff
@@ -156,6 +156,55 @@ const getFullAssetBalance = (
   }, '0')
 }
 
+const getAccountsForSelectedAsset = (
+  selectedAsset: BraveWallet.BlockchainToken,
+  accounts: BraveWallet.AccountInfo[]
+) => {
+  if (selectedAsset.chainId === BraveWallet.BITCOIN_MAINNET) {
+    return accounts.filter((account) =>
+      [
+        BraveWallet.KeyringId.kBitcoin84,
+        BraveWallet.KeyringId.kBitcoinImport
+      ].includes(account.accountId.keyringId)
+    )
+  }
+  if (selectedAsset.chainId === BraveWallet.BITCOIN_TESTNET) {
+    return accounts.filter((account) =>
+      [
+        BraveWallet.KeyringId.kBitcoin84Testnet,
+        BraveWallet.KeyringId.kBitcoinImportTestnet
+      ].includes(account.accountId.keyringId)
```

Comment:
should we extract this array so it can be reused and so that it is only created once?
```suggestion
      BITCOIN_TESTNET_KEYRING_IDS.includes(account.accountId.keyringId)
```

Reply:
Extracted!